### DISCUSSION TITLE
Too few arguments to function Filament\Tables\Table::getRecordAction()

### DISCUSSION THREAD

**Discussion by saidhappy010** [filamentphp/filament#4031] [Help]
I am using filament table in a project for the first time.
by following the instructions in the documentation,
but unfortunately I get this error,
`Too few arguments to function Filament\Tables\Table::getRecordAction(), 0 passed in /var/www/storage/framework/views/58d3a866489d6c130434c151d5797f5e503d5cc1.php on line 548 and exactly 1 expected`
how can i fix this problem.
Thanks

**Comment by danharrin** [comment]
Does your table have a primary key? Send your `getTableQuery()`
